### PR TITLE
Fixed bug unable to deactivate counterparty

### PR DESCRIPTION
**Pull request by AlexeyShok**
Fixed bug: unable to deactivate counterparty

### CODE REVIEW

File: Api/AdministratorServices/CounterpartyManagementService.cs [hunk 1]
```diff
@@ -231,9 +231,17 @@ async Task ChangeCounterpartyAccountsActivityStatus()
...
+                if (displacedMaster == default)
+                {
+                    var (_, isFailure, currentMaster, error) = await GetRootAgencyMasterAgent(counterparty.Id);
+                    if (isFailure)
+                        return Result.Failure(error);
+
+                    master = currentMaster;
+                }
+                else
+                    master = displacedMaster;
```
Comment (kirillta) [inline Api/AdministratorServices/CounterpartyManagementService.cs:244]: If one branch surrounded by braces, the other one must be surrounded too

Comment (kirillta) [reply Api/AdministratorServices/CounterpartyManagementService.cs:244]: Also, could we assign `displacedMaster` to `master` at the beginning of the method and remove the else branch?

Comment (AlexeyShok) [reply Api/AdministratorServices/CounterpartyManagementService.cs]: Fixed

**Review by kirillta — APPROVED** [review]

**Review by mingazhev — APPROVED** [review]
Not perfect but let's merge for now

**Pull request merged**